Tests/StreamChatUITests/SnapshotTests/ChatChannel/ChatChannelVC_Tests.swift

```diff
@@ -186,6 +186,26 @@ final class ChatChannelVC_Tests: XCTestCase {
         )
     }
 
+    func test_whenShouldMessagesStartAtTheTopEnabled() {
+        var components = Components.mock
+        components.shouldMessagesStartAtTheTop = true
+        vc.components = components
+
+        channelControllerMock.simulateInitial(
+            channel: .mock(cid: .unique),
+            messages: [
+                .mock(id: .unique, cid: .unique, text: "Hello", author: .mock(id: .unique)),
+                .mock(id: .unique, cid: .unique, text: "Cool", author: .mock(id: .unique))
+            ],
+            state: .localDataFetched
+        )
+
+        AssertSnapshot(
+            vc,
+            variants: [.smallDark] // This variant is the only one where the contentInset is stable between tests
```

Comment:
What about `smallLight`? The contentInset shouldn't be different depending on darkMode, right?
Additionally, do you mind elaborating what's the issue with the other variants and we can't use them?

Reply:
Sometimes, the contentInset does not remain 100% in the same position after the adjustment, and even in the demo app, we can see some minor little jumps when this happens. It looks like for the snapshot tests, this also happens, the adjustment is not 100% the same, which causes the snapshot test to fail between tests. I'm not sure why with small font, this does not happen, but it is what I found while trying it out.

Reply:
I see. Not sure if that requires further investigation to understand what's going on there. Maybe also an increased tolerance for the image comparison can help mitigate the issue and allow us to test on all variants.

Reply:
Unfortunately that does not work very well, and it slows down the speed of the snapshot tests if we decrease the precision :/ If a customer complains about these minor jitters, we can further investigate 👍 Right now they are barely noticeable 👍